target/linux/generic/config-6.1

```diff
@@ -5053,6 +5076,7 @@ CONFIG_PWRSEQ_SIMPLE=y
 # CONFIG_RALINK is not set
 # CONFIG_RANDOM32_SELFTEST is not set
 # CONFIG_RANDOMIZE_BASE is not set
+CONFIG_RANDOMIZE_KSTACK_OFFSET=y
```

Comment:
Can you explain why you enable this? Just because it is the default value?

Reply:
Previously, support for the `randomize_kstack_offset` was unconditionally compiled in for supported arches (arm64, ppc, x86, s390), so this changes nothing but just exposes the previous support for the feature.

Default usage is still determined by `RANDOMIZE_KSTACK_OFFSET_DEFAULT`, which is disabled both in kernel 5.15 and 6.1, with zero performance overhead. See https://lore.kernel.org/lkml/20220128114446.740575-1-elver@google.com/.